src/styles/blocks/antisnake.scss

```diff
@@ -0,0 +1,34 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($height, 1fr));
+  grid-auto-rows: $height;
+
+  @media (min-width: ($height * 4)) {
+    grid-template-columns: repeat(3, minmax($height, 1fr));
+  }
+
+}
```

Comment:
Some empty spaces, fix them

Reply:
Thanks a lot for your review! Empty spaces were removed